### DISCUSSION TITLE
Unable to enroll our university

### DISCUSSION THREAD

**Discussion by maqsudbek** [community/community#128653] [GitHub Education] [answered]
### Hi everyone,

We have applied to enroll our university to github education school partner program. We have uploaded all necessary information, and it was almost 2 month ago, still it is showing "Pending" in the page. When I tried to submit again, I get "404" page.
![image](https://github.com/community/community/assets/3115855/11e1c821-e407-4853-afd7-bf27191281b3)

what should we do ?

**Comment by yilmazemrecom** [comment]
You can open support from here. https://docs.github.com/en/support/contacting-github-support

**Comment by phoebequincy** [comment, marked as answer]
Hi [maqsudbek](https://github.com/maqsudbek) - please do [submit a ticket](https://support.github.com/contact/campus-program). Can you share the name of the school? (That will help us route your ticket more quickly.)
Thanks kindly,
Phoebe Quincy
GitHub Education

**Reply by maqsudbek** [reply]
Thanks for the answer. I opened a ticket (#2845206). 
Name of our university: "Turin Polytechnic University in Tashkent", located in Tashkent, Uzbekistan.
